### DISCUSSION TITLE
ability to override hostname in webhook or email alert link

### DISCUSSION THREAD

**Discussion by nycalex** [jhuckaby/Cronicle#737] [Ideas]
Good day,

It could be very useful to have an ability to override the hosname that's sent as a  part of link in the  webhook or email alert.

In case of a deployment behind a reverse proxy, users access an alternative public hostname, not the internal hostname known to the server itself.  Therefore the link in the alerts is pointing at a un-resolvable internal hostname.

thanks!!!

**Comment by jhuckaby** [comment]
https://github.com/jhuckaby/Cronicle/blob/master/docs/Configuration.md#base_app_url

### base_app_url

This should be set to a fully-qualified URL, pointing to your Cronicle server, including the HTTP port number if non-standard.  Do not include a trailing slash.  This is used in e-mails to create self-referencing URLs.  Example:

```
http://local.cronicle.com:3012
```

If you are running Cronicle behind a load balancer, this should be set to the load balanced virtual hostname.

**Comment by nycalex** [comment]
does the Cronicle app itself use base_app_url to access itself? I would think so, otherwise the setup would not be so sensitive to getting this field set up right.

In case I'm describing, there's also authentication involved in front-ending the  Cronicle with a reverse proxy, kind of like a captive portal. So the Cronicle front end is only served to me after authentication page on the proxy.

So it Cronicle app would not be able to access itself via such external name.
